automation/jleechanorg_pr_automation/tests/test_packaging_integration.py

```
@@ -75,6 +75,8 @@ def test_automation_safety_manager_module_execution_has_no_runpy_warning(self):
             ],
             capture_output=True,
             text=True,
+            cwd=os.path.dirname(os.path.dirname(jleechanorg_pr_automation.__file__)),
+            env={**os.environ, "PYTHONPATH": os.path.dirname(os.path.dirname(jleechanorg_pr_automation.__file__))},
```

Ruff (0.15.5)

[warning] `os.path.dirname()` should be replaced by `Path.parent`

(PTH120)